README.md

```diff
@@ -60,20 +60,36 @@ copy the WORKSPACE snippet into your `WORKSPACE` file.
 
 ## Usage
 
-One-time re-format all files:
+⚡ One-time re-format all files:
```

Comment:
⚡ fast?

Reply:
I dunno, it was just boring-looking so I needed an emoji. 💎 maybe?